Update imported filtering capabilities (#509)
plausible/docs: docs/google-analytics-import.md

### Filtering 

Google Analytics data is imported using Google API which has limits on the number of dimensions that can be fetched in a single query. This means we can't fetch entries that contain all the dimensions we're interested in and instead need to fetch data for each dimension individually.

> This means that we're unable to look at how most dimensions interact with each other, making [our filtering capability](filters-segments.md) limited with imported data. For example, we're unable to tell how many visitors with a specific page came from a specific source - this data does not exist. The same happens when we're filtering by a property and breaking down by another.

Reviewer: ```suggestion
This means that we're unable to look at how most dimensions interact with each other, making [our filtering capability](filters-segments.md) limited with imported data. For example, we're unable to tell how many visitors with a specific page came from a specific source - this data does not exist. The same happens when we're filtering by one property and breaking down by another.
```

> There are several exceptions to this though, because some properties are imported in groups. The properties in the following groups can be filtered at the same time:

Reviewer: ```suggestion
There are several exceptions to this though, because some properties are aggregated and grouped together. The properties in the following groups can be filtered at the same time:
```

* Countries, regions, cities
* Operating systems and their versions

> * Hostname and page

Reviewer: ```suggestion
* Hostnames and pages
```
